lib/matplotlib/testing/__init__.py

```diff
@@ -105,6 +105,9 @@ def subprocess_run_for_testing(command, env=None, timeout=60, stdout=None,
             import pytest
             pytest.xfail("Fork failure")
         raise
+    except subprocess.CalledProcessError as e:
+        import pytest
+        pytest.xfail(f"Subprocess failed with exit code {e.returncode}:\n{e.stdout}\n{e.stderr}")
```

Comment:
This should be a plain fail not an xfail, also made it a bit more verbose to make stdout and stderr output easier to read:

```suggestion
        pytest.fail(
            f"Subprocess failed with exit code {e.returncode}:\n\n"
            f"STDOUT:\n{e.stdout}\n\nSTDERR:\n{e.stderr}")
```

Reply:
Thanks! I agree I like yours better. Just made the change.